apps/vr-tests-react-components/src/stories/Charts/DonutChart.stories.tsx

```diff
@@ -18,8 +18,8 @@ export default {
 
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33323&commitId=674e00cabe51e224d71e269fc7250c86b14e5191&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|

</details>


  <details><summary><b>Charts-DonutChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Charts-DonutChart.Basic RTL.chromium.png|189| Changed|
|Charts-DonutChart.Dynamic RTL.chromium.png|266| Changed|
|Charts-DonutChart.Basic.chromium.png|189| Changed|
|Charts-DonutChart.Dynamic Dark Mode.chromium.png|225| Changed|
|Charts-DonutChart.Dynamic.chromium.png|267| Changed|
|Charts-DonutChart.Basic Dark Mode.chromium.png|175| Changed|

</details>


  <details><summary><b>Charts-HorizontalBarChart</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Charts-HorizontalBarChart.Basic.chromium.png|3975| Changed|
|Charts-HorizontalBarChart.Variant RTL.chromium.png|54| Changed|
|Charts-HorizontalBarChart.Basic Dark Mode.chromium.png|10637| Changed|
|Charts-HorizontalBarChart.Variant Dark Mode.chromium.png|24| Changed|
|Charts-HorizontalBarChart.With Benchmark Dark Mode.chromium.png|7215| Changed|
|Charts-HorizontalBarChart.With Benchmark RTL.chromium.png|2678| Changed|
|Charts-HorizontalBarChart.Basic RTL.chromium.png|3975| Changed|
|Charts-HorizontalBarChart.Variant.chromium.png|54| Changed|
|Charts-HorizontalBarChart.With_Benchmark.chromium.png|2678| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1137| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>